decentraland/unity-explorer
# feat: added profile pooling

## What does this PR change?

Implements profile pooling. Profiles will always be created through a pool and the disposal will release the instance.

TODO: make proper release of peers that are no longer around the player

## How to test the changes?

1. Launch the explorer
2. Check this report still working: https://github.com/decentraland/unity-explorer/issues/792
3. Check that everytime you change wearables or emotes your avatar is updated in-world
4. Check other peers avatars, they should be propagated as expected
5. Logout->login with new account and/or same account. Observe your avatar is properly updated

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
